lib/optimizely/decision_service.rb

```diff
@@ -146,7 +145,7 @@ def get_variation_for_feature_experiment(feature_flag, user_id, attributes = nil
       # user_id - String ID for the user
       # attributes - Hash representing user attributes
       #
-      # Returns a hash with the experiment and variation where visitor will be bucketed
+      # Returns decision with the experiment and variation where visitor will be bucketed
```

Comment:
Please also update the method return comment for
getVariationForFeatureRollout. I also pointed out in the last review